client/flutter/divkit/example/lib/src/pages/testing/testing.dart

```diff
@@ -0,0 +1,169 @@
+}
+
+class _TestingPage extends State<TestingPage> {
+  final MenuController menuController = MenuController();
+  Set<String> selectedTags = {};
+
+  void toggleTag(String tag) {
+    setState(() {
+      if (selectedTags.contains(tag)) {
+        selectedTags.remove(tag);
+      } else {
+        selectedTags.add(tag);
+      }
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) => FutureBuilder<List<Item>>(
+        future: loadList(),
+        builder: (_, snapshot) {
+          if (snapshot.hasError) {
+            return Center(
+              child: Text(snapshot.error.toString()),
+            );
+          }
+
+          if (snapshot.hasData) {
+            final data = snapshot.data
+                ?.where(
+                  (element) =>
+                      selectedTags.isEmpty ||
+                      element.tags.intersection(selectedTags).isNotEmpty,
+                )
+                .toList();
+
+            if (data != null) {
+              final tags = snapshot.data! //
+                  .map((e) => e.tags)
+                  .expand((e) => e)
+                  .toSet();
+
+              return Scaffold(
+                appBar: AppBar(
+                  title: const Text("Testing"),
+                  actions: [
+                    MenuAnchor(
+                      controller: menuController,
+                      menuChildren: [
+                        for (final tag in tags) //
+                          MenuItemButton(
+                            closeOnActivate: false,
+                            onPressed: () => toggleTag(tag),
+                            leadingIcon: Icon(
+                              selectedTags.contains(tag)
+                                  ? Icons.check_box
+                                  : Icons.check_box_outline_blank,
+                            ),
+                            child: Text(tag),
+                          ),
+                      ],
+                      child: IconButton(
+                        onPressed: () => menuController.open(),
+                        icon: const Icon(Icons.tag),
+                      ),
+                    ),
+                  ],
+                ),
+                body: ListView.builder(
+                  padding: const EdgeInsets.all(16),
+                  itemCount: data.length,
+                  itemBuilder: (context, index) {
+                    final item = data[index];
+                    return FilledButton.tonalIcon(
+                      onPressed: () => Navigator.of(context).push(
+                        MaterialPageRoute(
+                          builder: (context) => ScenarioPage(
+                            scenarios: data,
+                            index: index,
+                          ),
+                        ),
+                      ),
+                      label: Text(item.title),
+                      iconAlignment: IconAlignment.end,
```

Comment:
We have a wide range of versions now and we have been supporting the Flutter SDK since 3.7.12.

So we cannot use this property.

Reply:
Done